### PR TITLE
Feat/issue 67 trust pid mdl nov7updates - ARF v1.2

### LINKED ISSUE #67: Incorporate changes regarding Trust Modes, PID and mDL Rule book - ARF v1.2
### Description
Incorporate the ARF text additions as per 
* https://github.com/eu-digital-identity-wallet/ec-internal-desk/issues/20

### Tasks 
1. [x] Update the copy (markdown) 
2. [ ] Mark new version `v1.2.0`

### Acceptance Criteria
The markdown document reflects the `v1.2.0` draft Word document 

### Acceptance Criteria
The following scenarios shall be covered.

### Resources
* [ARF v1.2.0 working draft](https://eceuropaeu.sharepoint.com/:w:/r/teams/GRP-GRP-CNECTR3-eID/Shared%20Documents/General/ARF/Issue%20%2367%20-%20v1.2.0/1.1%20and%201.2%20compared.docx?d=w712bf30b6a7d4d788cfd3ef3104100c1&csf=1&web=1&e=KRaZVp) (with track changes)

### PR DESCRIPTION
**Pull request by skounis**
ARF v1.2 

Resolves #67

### CODE REVIEW

File: docs/arf.md [hunk 1]
```diff
@@ -551,43 +561,43 @@ mentioned depending on the specific role and their criticality for
...
 to the registry information. The terms and conditions of entities to
 become registered are for each registrar to determine unless specified
-in e.g., sectoral rules.
+in e.g., EU law or sectoral rules.
 
-#### 4.1.5. Qualified Electronic Attestation of Attributes Providers
+#### 4.1.5 Qualified Electronic Attestation of Attributes Providers
 
-Qualified EAA are provided by QTSPs. The general Trust Framework for
+Qualified EAA are provided by QTSPs. The IDAS Regulation Trust Framework for
```
Comment (skounis) [inline docs/arf.md:572]: as per @duviner change `IDAS` to `eIDAS`

**Review by duviner — APPROVED** [review]

File: docs/arf.md (hunk 1 above)
Comment (skounis) [reply docs/arf.md]: Fixed in https://github.com/skounis/architecture-and-reference-framework/commit/80d00cf5ad1c3930235e4140b1fc8a975638f787

**Comment by stevenvegt** [comment]
Hi all, I was wondering about the proces around this PR. Why is it first approved and later closed without a comment? Is there another place where the discussion takes place? Thanks!

**Comment by skounis** [comment]
Hi @stevenvegt 

PRs are considered draft work and not final changes. During the review process, we discovered that further changes are needed so the text will be ready for publication. 

The [issue tracking space](https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/issues) in this repository is the place for any public discussion.

**Pull request closed without merging**